src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -79,7 +80,12 @@
                     "READ_PAIR_OPTICAL_DUPLICATES.\n" +
                     "\n" +
                     "Usage example: java -jar picard.jar MarkDuplicates I=input.bam \\\n" +
-                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n";
+                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n" +
+                    "\n" +
+                    "The program can take either coordinate-sorted or query-sorted input, however the behavior is slightly different." +
+                    "When the input is coordinate-sorted, unmapped mate of mapped records, and supplementary and secondary alignments are not" +
+                    "marked as duplicates. When the input is query-sorted (actually query-grouped) then unmapped mates get marked as their mapped" +
+                    "counter-parts, and secondary and supplementary reads get marked as the primary records with the same query-name.";
```

Comment:
this line and the few above need either a newline or a space at the end of each string.